README.md

```diff
@@ -4,6 +4,7 @@ Currently, we support the following AUR helpers:
 
 - pacaur (recommended, default)
 - yaourt
+- yay
```

Comment:
Please mention lack of auronly support

Reply:
(also, please consider filing a bugs with yay people about lack of support for AUR only upgrades, and also that they silently ignore wrong flags)